arduino-ide-extension/src/browser/library/library-list-widget.ts

```diff
@@ -201,7 +201,8 @@ class MessageBoxDialog extends AbstractDialog<MessageBoxDialog.Result> {
       options.buttons || [nls.localize('vscode/issueMainService/ok', 'OK')]
     ).forEach((text, index) => {
       const button = this.createButton(text);
-      button.classList.add(index === 0 ? 'main' : 'secondary');
+      const i = options.buttons ? options.buttons?.length - 1 : 0;
+      button.classList.add(index === i ? 'main' : 'secondary');
```

Comment:
This code works fine, but for readability reasons I would prefer to see something like this:
```suggestion
      const isPrimaryButton = index === options.buttons?.length - 1 : 0;
      button.classList.add(isPrimaryButton ? 'main' : 'secondary');
```
What do you think? @francescospissu

Reply:
I think a piece of code is missing in your suggestion because in this way it couldn't work.

Do you mean something like this?

```javascript
const isPrimaryButton = index === (options.buttons ? options.buttons?.length - 1 : 0);
button.classList.add(isPrimaryButton ? 'main' : 'secondary');
```

Reply:
You're right, I made a mistake. This is what I wanted to suggest actually:
```
const isPrimaryButton = index === options.buttons?.length - 1;
button.classList.add(isPrimaryButton ? 'main' : 'secondary');
```
This should work the same – although I haven't tried it – and for me it's easier to read.
But your latest suggestion works as well for me, whatever you prefer.

Reply:
I've tried it before but haven't used it because but this error is thrown:

```
Object is possibly 'undefined'.
```

I've added the other change [here](https://github.com/arduino/arduino-ide/pull/1382/commits/eb37f689cce9db34f60297962c58f8ec218d98f3).